diff -up bro-20080804/src/X509.cc.openssl bro-20080804/src/X509.cc
--- bro-20080804/src/X509.cc.openssl	2006-10-07 04:20:48.000000000 +0200
+++ bro-20080804/src/X509.cc	2009-08-25 12:28:10.000000000 +0200
@@ -191,7 +191,7 @@ int X509_Cert::verifyChain(Contents_SSL*
 	// but in chain format).
 
 	// Init the stack.
-	STACK_OF(X509)* untrustedCerts = sk_new_null();
+	STACK_OF(X509)* untrustedCerts = sk_X509_new_null();
 	if ( ! untrustedCerts )
 		{
 		// Internal error allocating stack of untrusted certs.
@@ -232,7 +232,7 @@ int X509_Cert::verifyChain(Contents_SSL*
 		else
 			// The remaining certificates (if any) are put into
 			// the list of untrusted certificates
-			sk_push(untrustedCerts, (char*) pTemp);
+			sk_X509_push(untrustedCerts, pTemp);
 
 		tempLength += certLength + 3;
 		}
@@ -258,7 +258,7 @@ int X509_Cert::verifyChain(Contents_SSL*
 	// Free the stack, incuding. contents.
 
 	// FIXME: could this break Bro's memory tracking?
-	sk_pop_free(untrustedCerts, free);
+	sk_X509_pop_free(untrustedCerts, ((void (*)(X509 *))free));
 
 	return ret;
 	}
