--- bro-2.5.1/CMakeLists.txt	2017-09-29 23:44:44.898323711 -0500
+++ bro-2.5.1/CMakeLists.txt.new	2017-09-29 23:48:50.975141692 -0500
@@ -173,7 +173,7 @@
 set(DEFAULT_BROPATH .:${BRO_SCRIPT_INSTALL_PATH}:${BRO_SCRIPT_INSTALL_PATH}/policy:${BRO_SCRIPT_INSTALL_PATH}/site)

 if ( NOT BINARY_PACKAGING_MODE )
-    set(BRO_DIST ${CMAKE_SOURCE_DIR})
+    set(BRO_DIST ${CMAKE_SOURCE_DIR} CACHE STRING "Path for Bro development distribution files." )
 endif ()

 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/bro-config.h.in
